Datahub.Web/Pages/Asset.cshtml.cs

```diff
@@ -12,18 +12,23 @@ namespace Datahub.Web.Pages
     public class AssetModel : PageModel
     {
         private readonly IHostingEnvironment _env;
-        public AssetModel(IHostingEnvironment env)
+        private readonly IDynamodbService _db;
+
+        public AssetModel(IHostingEnvironment env, IDynamodbService db)
         {
             _env = env;
+            _db = db;
         }
 
         public Asset Asset { get; set; }
 
 
         public async Task OnGetAsync(string assetId)
         {
-            var assets = await JsonLoader.LoadAssets(_env.ContentRootPath);
-            this.Asset = assets.Single(a => a.Id == assetId);
+            this.Asset = await _db.GetAsset(assetId);
```

Comment:
Could we switch on whether the DB_TABLE had been defined, and if not, use the existing dev JSON data?

Reply:
Resolved 79986d48d46bf09c02e1c22f14aae776019fedf6